clients/drcachesim/tracer/tracer.cpp

```diff
@@ -121,11 +121,19 @@ static size_t max_buf_size;
 
 static drvector_t scratch_reserve_vec;
 
+// For virtual-to-physical markers, we need extra space to insert them after
+// filling up a buffer.  We do not expect many per trace buffer as we only emit
+// them for never-before-seen pages..
+// TODO i#4014: Split the buffer if we hit this limit.  For now, our markers are
+// best-effort and we just drop them if this limit is exceeded.
+static constexpr int MAX_V2P_ENTRIES = 128;
```

Comment:
So all the 128 accesses will have to be to a different page for entries to be dropped right? I guess that's okay.

Reply:
How about setting it based on `MAX_NUM_ENTRIES`, since they directly relate to each other.

Or atleast add a comment by the definition of `MAX_NUM_ENTRIES` so that we remember to increase `MAX_V2P_ENTRIES` if we increase `MAX_NUM_ENTRIES`.

Reply:
The plan is to implement the buffer splitting, since a larger max than I'm comfortable allocating (this per-thread heap adds up), so such a comment would be removed at that time.  But I added it and hopefully will notice and remove it.

Reply:
Left off *a larger max...is needed": some local tests need 512 instead of 128.